doc/source/whatsnew/v0.19.0.rst

```diff
@@ -1236,6 +1236,7 @@ Operators now preserve dtypes
 - Sparse data structure now can preserve ``dtype`` after arithmetic ops (:issue:`13848`)
 
 .. ipython:: python
+   :okwarning:
```

Comment:
maybe better to make these code blocks?

Reply:
Probably. I'm still hoping someone will write a script to do this automatically someday. Just capture the output and freeze that.

Reply:
lol

Reply:
> I'm still hoping someone will write a script to do this automatically someday. Just capture the output and freeze that.

That's also my long term plan ;) 
It sounds as not too hard, but sphinx keeps being a mystery most of the time ..